src/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Demo from './cardHolder'
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
I think it's better to keep `App.js` cause I saw you deleted it? Put `<Demo/>` to be returned in `<App/>`, and keep `<App/>` as a main component to be rendered in `index.js`. The reason is usually when our application grows with more components and stuffs, we will most likely encapsulate them in `<App/>` with all the routes

Reply:
Oh!! I see! ok got it